### DISCUSSION TITLE
Task View not working with Windows 10 Taskbar

### DISCUSSION THREAD

**Discussion by lesonion** [valinet/ExplorerPatcher#1099] [General]
So I updated to the latest version of ExplorerPatcher and quite early realised that the Win-Tab shortcut for TaskView does not work. In turn, trying to use this shortcut also causes normal Alt-Tab not to work either. When restarting File Explorer normal Alt-Tab works, but as soon as I try Win-Tab it causes Alt-Tab not to work again. After trying different windows fixes I went back to changing different settings in ExplorerPatcher and quickly realised that the problem was using Windows 10 Taskbar style. Switching to Windows 11 taskbar style directly fixed the problem. The problem didn't occur until today when I updated to version 22621.1344.53.1 of ExplorerPatcher. 

If this is anything on my end that I can use to fix this I'm looking for anyone to help me. Otherwise this is for the devs so they know the issue exists.

**Comment by AlexAdasCca** [comment]
See the workaround here: https://github.com/valinet/ExplorerPatcher/discussions/1087

**Reply by lesonion** [reply]
Thanks for the answer, didn't hit me that I could look for discussions including Alt-Tab and Win-Tab and not just Task View. Fixed the problem for me
